app/api/main.py

```diff
@@ -0,0 +1,10 @@
+from fastapi import FastAPI
+from routers import anonymize_file
```

Comment:
it's more "pythonic" to import like that

```python
from routers.anonymize_file import router
```

Reply:
it was done that way cuz we may have several routers so can't assign for each router a unique name thus kept the parent file , tell me if you still wanna me change it @achab

Reply:
you can also rename `router` to `anonymize_router` in the file `app/api/routers/anonymize_file.py`. when possible, it's better to import using full path